sklearn/externals/_scipy_linalg.py

```diff
@@ -0,0 +1,116 @@
+"""Copyright (c) 2001-2002 Enthought, Inc.  2003-2019, SciPy Developers.
```

Comment:
please comment here that this should remain pinned to version 1.2, not upgraded like other externals.